nautobot/dcim/tests/test_api.py

```diff
@@ -695,6 +696,37 @@ def setUpTestData(cls):
         ]
 
 
+class HardwareFamilyTest(APIViewTestCases.APIViewTestCase):
+    model = HardwareFamily
+    create_data = [
+        {
+            "name": "Hardware Family 4",
+            "description": "Fourth Hardware Family",
+        },
+        {
+            "name": "Hardware Family 5",
+        },
+        {
+            "name": "Hardware Family 6",
+            "description": "Sixth Hardware Family",
+        },
+        {
+            "name": "Hardware Family 7",
+        },
+    ]
+    bulk_update_data = {
+        "description": "New description",
+    }
+
+    @classmethod
+    def setUpTestData(cls):
+        # FIXME: This has to be replaced with# `get_deletable_object` and
+        # `get_deletable_object_pks` but this is a workaround just so all of these objects are
+        # deletable for now.
+        Device.objects.all().delete()
+        DeviceType.objects.all().delete()
```

Comment:
Is this actually needed at present? What's the issue if we don't delete these - can we just not get a deletable HardwareFamily record(s) for the test?